runbot_gitlab/controllers/gitlab_ci_controller.py

```diff
@@ -71,3 +73,22 @@ def hook_gitlab(self, repo_id=None, **post):
                         forced_build.write({"deployv_image_built": True})
                         forced_build._logger(msg)
         return ""
+
+    @http.route(['/runbot/hook_github/org'], type='json', auth="public", methods=["POST"], website=True, csrf=False)
+    def hook_github(self):
+        data = request.jsonrequest
+        sha = data.get("workflow_job", {}).get("head_sha", False)
+        if not sha:
+            raise BadRequest("SHA is required")
+
+        build = request.env["runbot.build"].sudo().search([("name", "=", sha)])
+        if not build:
+            raise BadRequest("No build associated with this SHA has been found")
+
+        name = data["workflow_job"].get("name", "")
+        status = data["workflow_job"].get("status", "")
+        conclusion = data["workflow_job"].get("conclusion")
+
+        if name == "build_docker" and status == "completed" and conclusion == "success":
```

Comment:
`build_docker`? I think that's the name on GitLab, in GitHub it should be another one

Reply:
I based myself on the CI file for addons-vauxoo:

https://github.com/Vauxoo/addons-vauxoo/blob/5d2d0ef89eda4d1c9e575525f665c47bb3cc0d4d/.github/workflows/test.yml#L52

In which case AFAIK it is indeed build_docker,  but perhaps I am missing other projects? This one was the first I found on github with a CI pipeline setup.

Reply:
Oh, I see, then it seems it's correct.